app/models/user.rb

```diff
@@ -7,4 +7,8 @@ class User < ActiveRecord::Base
   validates :password, presence: true
 
   delegate :supported?, to: :location, allow_nil: true
+
+  def name
+    "first last"
```

Comment:
This is a bug / oversight. 

It will be addressed in the later phases of https://trello.com/c/1TzjekfA